src/Akeneo/Crowdin/Api/UploadTranslation.php

```diff
@@ -92,25 +93,26 @@ public function execute()
     }
 
     /**
-     * @param string $crowdinPath the Crowdin file path
-     * @param string $localPath   the local file path
+     * @param string $localPath
+     * @param string $crowdinPath
+     * @param string $exportPattern
+     * @param string $title
      *
-     * @throws InvalidArgumentException
-     *
-     * @return UploadTranslation
+     * @return $this
      */
-    public function addTranslation($crowdinPath, $localPath)
+    public function addTranslation($localPath, $crowdinPath, $exportPattern = null, $title = null)
```

Comment:
You're right, it's a massive BC Break, introduced to ensure consistency with following methods,
 - https://github.com/akeneo/php-crowdin-api/blob/master/src/Akeneo/Crowdin/Api/UpdateFile.php#L92
 - https://github.com/akeneo/php-crowdin-api/blob/master/src/Akeneo/Crowdin/Api/AddFile.php#L101